DESCRIPTION

```diff
@@ -17,7 +17,11 @@ Authors@R: c(
 	person("Hervé", "Pagès", role=c("aut", "cre"),
 		email="hpages.on.github@gmail.com"),
 	person("Valerie", "Obenchain", role="aut"),
-	person("Martin", "Morgan", role="aut"))
+	person("Martin", "Morgan", role="aut"),
+	person("Robert", "Castelo", role="ctb"
```

Comment:
I'm sorry, I approved a little too quickly. I noticed we're missing the closing parenthesis and comma for Robert's entry.